.gitignore

```
@@ -92,3 +92,5 @@ iOSInjectionProject/
 # Netowkring configurations like API keys
 
 NetworkConfiguration.swift
+*.xcuserstate
+NetflixClone/Classes/Scenes/Home/.DS_Store
```

```suggestion
.DS_Store
```
It's enough to ignore any file with `.DS_Store` name

---

Thanks🔥! Noted for the future.